src/manifest.ts

```diff
@@ -147,30 +224,40 @@ export class Manifest {
 
   protected async getPackagesToRelease(
     commits: Commit[],
-    lastReleaseSha?: string
+    sha?: string
   ): Promise<PackageReleaseData[]> {
     const packages = await this.readConfig();
-    const manifestVersions = JSON.parse(
-      (await this.getManifest(lastReleaseSha)).parsedContent
-    );
+    const [manifestVersions, atSha] = await this.getManifestVersions(sha);
     const cs = new CommitSplit({
       includeEmpty: true,
       packagePaths: packages.map(p => p.path),
     });
     const commitsPerPath = cs.split(commits);
     const packagesToRelease: Record<string, PackageReleaseData> = {};
     const missingVersionPaths = [];
+    const defaultBranch = await this.gh.getDefaultBranch();
     for (const pkg of packages) {
       const commits = commitsPerPath[pkg.path];
       if (!commits || commits.length === 0) {
         continue;
       }
-      const lastVersion = manifestVersions[pkg.path];
+      const lastVersion = manifestVersions.get(pkg.path);
       if (!lastVersion) {
+        this.checkpoint(
+          `Failed to find version for ${pkg.path} in ` +
+            `${this.manifestFileName} at ${atSha} of ${defaultBranch}`,
+          CheckpointType.Failure
+        );
         missingVersionPaths.push(pkg.path);
+      } else {
+        this.checkpoint(
+          `Found version ${lastVersion} for ${pkg.path} in ` +
+            `${this.manifestFileName} at ${atSha} of ${defaultBranch}`,
+          CheckpointType.Success
+        );
       }
       const releaserOptions = {
-        monorepoTags: packages.length > 1,
```

Comment:
That seems like the right call to me 100%. I'd honestly like us to move to the stable branch name across the board, because it would make it easier to back port to LTS branches.